namespace: dxl_manager
serial_ports:
    pi_out_port:
        port_name: "/dev/ttyAMA0"
        baud_rate: 1000000
        min_motor_id: 1
        max_motor_id: 20
        update_rate: 20
